_extensions/quarto-journals/plos/filter.lua

```
@@ -0,0 +1,81 @@
+              -- Remove header
+              return {}
+            end,
+            Para = function(el)
+              if (paraNum == 0) then
+                -- first paragraph is title sentence
+                if (el.content and el.content[1].t ~= "Strong") then
+                  el.content = pandoc.Inlines(pandoc.Strong(el.content))
+                end
+                el.content:insert(1, pandoc.RawInline('tex', '{'))
+                el.content:insert(pandoc.RawInline('tex', '}'))
+              elseif (paraNum == 1) then
```

Luacheck (1.2.0)

[warning] empty if branch

(W542)